.gitignore

```diff
@@ -10,3 +10,5 @@ dist/
 fbpic.egg-info/
 .cache/
 .eggs/
+
+.idea/
```

Comment:
Thanks for the PR! Could you revert the above changes to the `.gitignore`? Thanks!

Reply:
Will do.